dev-bin/make-man-pages.pl

```diff
@@ -19,6 +19,12 @@ sub main {
         "$build_dir/GeoIP.conf.md",
         "$build_dir/GeoIP.conf.5",
     );
+    _make_man(
+        'GeoIP.env',
+        9,
```

Comment:
I just picked a number 😬

Reply:
Is 9 even a thing? I think there's conventions around this. 5 probably is fine.